java/debugger.jpda/src/org/netbeans/modules/debugger/jpda/models/ObjectTranslation.java

```diff
@@ -273,5 +345,16 @@ public static ObjectTranslation createThreadTranslation(JPDADebuggerImpl debugge
     public static ObjectTranslation createLocalsTranslation(JPDADebuggerImpl debugger) {
         return new ObjectTranslation(debugger, LOCALS_ID);
     }
-    
+
+    private static final class TranslationFuture extends CompletableFuture<Object> {
+
+        @Override
+        public Object get() {
+            try {
+                return super.get();
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It looks like you are using CompletableFuture.get() without setting its timeout parameter. Be sure to set timeout values or handle timeout exceptions to prevent threads from waiting for a process to end. This can lead to downtime for your customers. [here](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/CompletableFuture.html#get-long-java.util.concurrent.TimeUnit-).